models/src/main/thrift/appsRendering.thrift

```diff
@@ -168,6 +168,8 @@ struct Newsletter {
     4: required string description
     5: required string frequency
     6: required string successDescription
+    7: optional string regionFocus
+    8: required boolean paused
```

Comment:
How do you see the `paused` flag being used by AR? I'm cautious about adding a new `required` field but it depends case-by-case

Reply:
We'd like the sign-up pages to be able to indicate if the Newsletter is currently paused (both on AR and DCR) - at the moment for DCR, frontend excludes the 'promotedNewsletter' data from the request if the newsletter is pause, so the sign-up form isn't included. For example - lab notes is paused and has custom content in the standfirst to explain that: 
https://www.theguardian.com/science/2016/jun/07/sign-up-for-lab-notes-the-guardians-weekly-science-update

It's not known exactly what the UX for paused newsletters would be - potentially we might want to still promote sign-ups even if the newsletter is paused. EG we might have a newsletter covering elections or the football season that would regularly be paused and unpaused.

We could probably make `paused` optional here  - its required in the Newsletters API, so would actually always be set, but downstream, we could just treat `undefined` as implying `false`.